meter.go

```diff
@@ -140,6 +158,9 @@ func (m *StandardMeter) Count() int64 {
 func (m *StandardMeter) Mark(n int64) {
 	m.lock.Lock()
 	defer m.lock.Unlock()
+	if m.stopped {
+		panic("Mark called on a stopped Meter")
```

Comment:
and change the interface for that? That's like the Mark() on a snapshot which also panics